src/plugins/inspector/public/plugin.tsx

```diff
@@ -59,9 +59,9 @@ export interface Start {
 export class InspectorPublicPlugin implements Plugin<Setup, Start> {
   views: InspectorViewRegistry | undefined;
 
-  constructor(initializerContext: PluginInitializerContext) {}
+  constructor(_initializerContext: PluginInitializerContext) {}
```

Comment:
Prepend parameter with `_` to dismiss the "unused variable" warning that shows in my editor.